pkg/util/util.go

```diff
@@ -59,3 +62,22 @@ func NamespaceOpenShiftObject(componentName string, applicationName string) (str
 	// Return the hyphenated namespaced name
 	return fmt.Sprintf("%s-%s", componentName, applicationName), nil
 }
+
+// Reads file path form URL file:///C:/path/to/file to C:\path\to\file
+func ReadFilePath(u *url.URL) string {
```

Comment:
We should add tests for this.. but we can ignore this for now just so we can fix the commands on Windows for the time being. Can you add a TODO: Write tests comment?

Reply:
sure, I'll look if I can add tests, to make PR checks back to green, if I fail, then adding `TODO` is my backup plan.